api/v1/manager_types.go

```diff
@@ -25,6 +27,65 @@ type ManagerSpec struct {
 	// Deprecated. Please use the Authentication CR for configuring authentication.
 	// +optional
 	Auth *Auth `json:"auth,omitempty"`
+
+	// Template describes the Manager Deployment pod that will be created.
+	// +optional
+	Template *ManagerDeploymentPodTemplateSpec `json:"template,omitempty"`
```

Comment:
I think this isn't following the other examples of this pattern quite right. e.g.,: https://github.com/tigera/operator/blob/ddeed95e5f6e1f1f3574c2b46def5c7660c1257d/api/v1/tenant_types.go#L88-L89

I think this should be `ManagerDeployment *ManagerDeployment`